sqlx/lexer.ts

```diff
@@ -182,50 +223,61 @@ class SqlxParseState {
   }
 }
 
+function getValueMappings() {
+  const valueMappings: {
```

Comment:
why not just a top level exported const? You'll get a nicer type too

Reply:
because sadly, java/typescript don't let you instantiate a top-level object whose keys are fields of other objects.